v2/pkg/catalog/loader/loader.go

```diff
@@ -195,8 +203,14 @@ func areWorkflowOrTemplatesValid(store *Store, filteredTemplatePaths map[string]
 				areTemplatesValid = false
 			}
 		} else {
+			if old, ok := uniqueTemplatesIDValidateMap[template.ID]; !ok {
```

Comment:
I'd rather use more self explaining variable names like `existingTemplatePath` instead of `old` and `found` instead of `ok`